src/io/flutter/actions/FlutterSdkAction.java

```diff
@@ -88,11 +85,14 @@ public void actionPerformed(AnActionEvent event) {
   public final void perform(@NotNull FlutterSdk sdk,
                             @NotNull Project project,
                             @Nullable AnActionEvent event,
-                            @SuppressWarnings("SameParameterValue") boolean logAction)
+                            @SuppressWarnings("SameParameterValue") boolean sendAnalytics)
     throws ExecutionException {
-    if (logAction) {
-      sendActionEvent();
+    if (sendAnalytics) {
+      FlutterInitializer.sendAnalyticsAction(this);
     }
+
+    FileDocumentManager.getInstance().saveAllDocuments();
```

Comment:
Pretty sure we want to make sure this is run on the event dispatch thread.  IOW wrap in

    ApplicationManager.getApplication().runWriteAction( ... )

?